components/card/card.html

```diff
@@ -0,0 +1,198 @@
+                        </div>
+
+                    </div>
+                </div>
+
+            </div>
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=one-dark&wt=none&l=ruby&width=703.75&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card-container%2522%253E%250A%253Cdiv%2520class%253D%2522img-contain%2522%253E%250A%253Cimg%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F400%2522%2520alt%253D%2522Photo%2522%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522card-details%2522%253E%250A%253Cspan%2520class%253D%2522tag%2522%253ENature%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522tag%2522%253ELake%253C%252Fspan%253E%250A%253Cdiv%2520class%253D%2522name%2522%253ECard%2520Vertical%253C%252Fdiv%253E%250A%253Cp%253ELorem%2520ipsum%2520dolor%2520sit%2520amet%252C%2520consectetur%2520adipiscing%2520elit%2520Consecteturb%2520sodales%2520morbi%2520dignissim%2520sed%2520diampharetra%2520vitae%2520ipsum%2520odio.%253C%252Fp%253E%250A%253Cdiv%2520class%253D%2522btn-card-details%2522%253E%250A%253Cbutton%2520class%253D%2522btn-primary%2520btn-outline-secondary%2520btn-card%2522%253EAdd%2520to%2520cart%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn-primary%2520btn-outline-secondary%2520btn-card%2522%253EBuy%2520now%253C%252Fbutton%253E%250A%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522card-container%2522%253E%250A%253Cdiv%2520class%253D%2522img-contain%2522%253E%250A%253Cimg%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F400%2522%2520alt%253D%2522Photo%2522%253E%250A%253Cdiv%2520class%253D%2522tag-badge%2522%253E%253Ci%2520class%253D%2522fas%2520fa-times-circle%2522%253E%253C%252Fi%253E%253Cdiv%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522card-details%2522%253E%250A%253Cspan%2520class%253D%2522tag%2522%253ENature%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522tag%2522%253ELake%253C%252Fspan%253E%250A%253Cdiv%2520class%253D%2522name%2522%253ECard%2520Vertical%253C%252Fdiv%253E%250A%253Cp%253ELorem%2520ipsum%2520dolor%2520sit%2520amet%252C%2520consectetur%2520adipiscing%2520elit.%2520Consectetur%2520sodales%2520morbi%2520dignissim%2520sed%2520diam%2520pharetra%2520vitae%2520ipsum%2520odio.%253C%252Fp%253E%250A%253Cdiv%2520class%253D%2522btn-card-details%2522%253E%250A%253Cbutton%2520class%253D%2522btn-primary%2520btn-outline-secondary%2520btn-card%2522%253EAdd%2520to%2520cart%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn-primary%2520btn-outline-secondary%2520btn-card%2522%253EBuy%2520now%253C%252Fbutton%253E%250A%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                style="width: 100%;
+  height: 335px;
+  border: 0;
+  transform: scale(1);
+  overflow: hidden;" sandbox="allow-scripts allow-same-origin">
+            </iframe>
```

Comment:
You can remove inline styling from here.

Reply:
Thank you for the suggestion ! Removed the inline styling